db/suspensionReasons.ts

```diff
@@ -0,0 +1,14 @@
+import { SuspensionReason } from '../types/SuspensionReason';
+import knexConfig from '../knexfile';
+import knex from 'knex';
+
+const db = knex(knexConfig);
+
+const getAllForPsychologist = async (psychologistId: string) : Promise<SuspensionReason[]> => db
+    .select()
+    .from('suspension_reasons')
```

Comment:
In another branch, we regrouped the table name in a dedicated file. We can add a TODO here to remember to use it here when both branches will be merged.